parrot/settings.py

```diff
@@ -5,7 +5,7 @@
 
 BASE_DIR = Path(__file__).parent.parent
 
-SECRET_KEY = '^c!x7!qejlut8=d3=mls9c%u-+(8t#&cfap5#=w%10t@&*b4=6'
+SECRET_KEY = env('SECRET_KEY')
```

Comment:
Prefix `PARROT_` is required for all app variables

```suggestion
SECRET_KEY = env('PARROT_SECRET_KEY')
```